responsibleai/tests/causal/test_causal_manager.py

```diff
@@ -48,6 +51,30 @@ def test_causal_save_and_load(self, housing_data, tmpdir):
         assert post_result.local_effects is not None
         assert post_result.policies is not None
 
+        # Remove the causal analaysis models to test the loading of
+        # causal models in case there is error in loading of the causal
+        # models.
+        all_causal_dirs = DirectoryManager.list_sub_directories(save_dir)
+        for causal_dir in all_causal_dirs:
+            dm = DirectoryManager(parent_directory_path=save_dir,
+                                  sub_directory_name=causal_dir)
+            causal_analysis_pkl_file_path = \
+                dm.get_data_directory() / "causal_analysis.pkl"
+            os.remove(causal_analysis_pkl_file_path)
+
+        model_load_err = ('ERROR-LOADING-EXPLAINER: '
```

Comment:
Error loading explainer in a causal test?

Reply:
Explainer is generic term. The warning is being raised from some generic code so chose explainer.